src/ProgressOnderwijsUtils/RetryDelayChooser.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace ProgressOnderwijsUtils
+{
+    /// <summary>
+    /// This class pick a retry delay based on the current error rate (see ErrorsPerDayToRetryDelay for the formula)

```

Comment:
Zin klopt niet

Reply:
Perhaps more descriptive on the purpose of this class

Reply:
updates.  Better?